Modules/clinic/posixmodule.c.h

```diff
@@ -5334,6 +5334,7 @@ PyDoc_STRVAR(os_pwritev__doc__,
 "\n"
 "- RWF_DSYNC\n"
 "- RWF_SYNC\n"
+"- RWF_APPEND\n"
```

Comment:
You must also update posixmodule.c docstring, and then run "make clinic" to *regenerate* this file.

Reply:
This was tried travis-ci seems to fail

Reply:
Please modify Modules/posixmodule.c and then run "make clinic".